cmd/soroban-cli/src/commands/contract/init.rs

```diff
@@ -261,7 +261,7 @@ impl Runner {
     }
 
     fn check_internet_connection() -> bool {
-        if let Ok(_req) = get(GITHUB_URL).call() {
+        if let Ok(_req) = http::blocking_client().get(GITHUB_URL).send() {
             return true;
         }
```

Comment:
Fyi this call is going to go away soon as part of:
- https://github.com/stellar/stellar-cli/issues/1586

Reply:
We can remove the blocking feature after it merged.